lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -334,6 +338,11 @@ def add_transaction_type(post, options)
         post[:a9] = options[:transaction_type] if options[:transaction_type]
       end
 
+      def add_processor(post, options)
+        post[:r1] = options[:processor] || 'CREDORAX'
```

Comment:
It looks like with the previous implementation, no ```r1``` parameter would have been set for a non-3DS transaction, whereas now ```r1``` will be set to ```CREDORAX``` for a non-3DS transaction that does not have the new ```processor``` option. This seems like it would be okay, but I'm not specifically familiar with the option. Is that difference fine?

Reply:
My understanding is that `r1` needs to be sent through for all transactions, not just 3DS transactions, whenever selecting a processor is enabled in the gateway settings. 

What I don't know (and sort of assume is okay) is the effect of sending `r1` when selecting a processor is *not* enabled in the gateway settings.